src/Razor/test/testapps/ComponentApp/ComponentApp.csproj

```diff
@@ -1,9 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
-  <PropertyGroup>
-    <RazorSdkDirectoryRoot>$(RazorSdkArtifactsDirectory)$(Configuration)\sdk-output\</RazorSdkDirectoryRoot>
-  </PropertyGroup>
```

Comment:
Is this a removal @JunTaoLuo could have done in one of his recent fixups?

Reply:
I believe so. I had to do that to get this testapp to work in my experimental VS instance.